### PR TITLE
added CRUD to userskillscontroller

### PR DESCRIPTION
**Pull request by kittymayo**

### CODE REVIEW

File: app/controllers/user_skills_controller.rb [hunk 1]
```diff
@@ -12,6 +21,21 @@ def create
...
+  end
+
+  def update
+    @user_skill = UserSkill.find(params[:id])
+    @user_skill.update(user_skills_params)
+  end
+
   private
 
   def user_skills_params
```
Comment (nonrevolver) [inline app/controllers/user_skills_controller.rb]: please populate this method :-)
```suggestion
  def user_skills_params
  $REQUIRE/PERMIT
  end
```

File: app/controllers/user_skills_controller.rb [hunk 2]
```diff
@@ -12,6 +21,21 @@ def create
     end
   end
 
+  def edit
+    @user_skill = UserSkill.find(params[:id])
```
Comment (TaniyaAmidon) [inline app/controllers/user_skills_controller.rb]: Please refactor params[:id] with user_skills_params method. Please move update method below the edit method

File: app/controllers/user_skills_controller.rb [hunk 3]
```diff
@@ -12,6 +21,21 @@ def create
...
+  end
+
+  def destroy
+    @user_skill = UserSkill.find(params[:id])
+    @user_skill.destroy
+    redirect_to user_show_path
+  end
+
+  def update
```
Comment (TaniyaAmidon) [inline app/controllers/user_skills_controller.rb]: please move update above destroy

File: app/controllers/user_skills_controller.rb [hunk 4]
```diff
@@ -1,6 +1,20 @@
 class UserSkillsController < ApplicationController
+  before_action :set_user_skill, only: [:show, :edit, :update, :destroy]
+
+  def index
+    @user_skills = UserSkill.all
+  end
+
+  def show
```
Comment (TaniyaAmidon) [inline app/controllers/user_skills_controller.rb]: please remove @user_skill

File: app/controllers/user_skills_controller.rb [hunk 5]
```diff
@@ -12,8 +26,23 @@ def create
     end
   end
 
+  def update
+    @user_skill
```
Comment (TaniyaAmidon) [inline app/controllers/user_skills_controller.rb]: please remove @user_skills

File: app/controllers/user_skills_controller.rb [hunk 6]
```diff
@@ -12,8 +26,23 @@ def create
...
+    @user_skill.update(user_skills_params)
+  end
+
+  def destroy
```
Comment (TaniyaAmidon) [inline app/controllers/user_skills_controller.rb]: please remove @user_skill

**Pull request merged**